terminal/linux.md

```diff
@@ -233,11 +233,131 @@ There are plenty of frameworks that have been created to help setup zsh configur
 
 3. To change to another prompt theme
 
-4. ```
+  ```
   $ prompt <name-of-the-theme>
   ```
 
 
+**Note:** If the above mentioned command does not work, follow the following steps to change the current prompt theme
+
+Go to your terminal window and type the following command:
+
+```
+nano ~/.zpreztorc
```

Comment:
forgot to show the prompt on this command. Check it on the other commands in this doc too